src/ga.js

```diff
@@ -1,5 +1,7 @@
 export function report(category, action, label) {
   if (isLocal()) {
+    console.log('Fake report: ');
+    console.log({ category, action, label });
```

Comment:
Are these supposed to be present? Seems like they were added for some kind of debugging.

Reply:
Yes they are for debugging reasons, but will trigger only in a local environment, I will let them there, thoughts @moshfeu ?

Reply:
Can keep that. Doesn't hurt. If it will, we'll remove it.